mmv1/third_party/terraform/services/securityposture/resource_securityposture_posture_deployment_test.go

```diff
@@ -69,12 +69,18 @@ resource "google_securityposture_posture" "posture_one" {
     }
 }
 
+resource "google_project" "posture_project" {
+  name       = "Posture Project"
+  project_id = "posture-project-id"
+  org_id     = "%{org_id}"
+}
```

Comment:
```suggestion
resource "google_project" "posture_project" {
  name       = "Posture Project"
  project_id = "tf-test-posture-project-id-%{random_suffix}"
  org_id     = "%{org_id}"
  deletion_policy = "DELETE"
}
```
We'll need `tf-test` prefix and random suffix for resource identifier. `deletion_policy = "DELETE"` needs to explicitly specified so that the project can be deleted after